railties/test/isolation/abstract_unit.rb

```diff
@@ -464,7 +464,7 @@ def use_frameworks(arr)
       $:.reject! { |path| path =~ %r'/(#{to_remove.join('|')})/' }
     end
 
-    def use_postgresql(multi_db: false)
+    def use_postgresql(multi_db: false, database_name: "railties")
```

Comment:
This seems like a reasonable solution to me.  What would be the downside of automatically using different database names, such as:

```suggestion
    def use_postgresql(multi_db: false, database_name: "railties#{"_multi" if multi_db}_#{Process.pid}")
```

and then returning `database_name` for when it's needed, e.g. in `test "db:prepare creates test database if it does not exist"`?

Reply:
Good idea, I've implemented your suggestion.